plugins/woocommerce/includes/admin/class-wc-admin-addons.php

```diff
@@ -1284,7 +1296,13 @@ public static function render_product_card( $data, $block_type = null ) {
 									);
 									?>
 								</span>
-								<span class="price-suffix"><?php esc_html_e( 'per year', 'woocommerce' ); ?></span>
+								<span class="price-suffix"><?php
+									$price_suffix = __( 'per year', 'woocommerce' );
+									if ( ! empty( $mapped->price_suffix ) ) {
+										$price_suffix = $mapped->price_suffix;
+									}
+									echo esc_html( $price_suffix );;
+									?></span>
```

Comment:
Can we do:

```suggestion
								<span class="price-suffix">
									<?php
									$price_suffix = __( 'per year', 'woocommerce' );
									if ( ! empty( $mapped->price_suffix ) ) {
										$price_suffix = $mapped->price_suffix;
									}
									echo esc_html( $price_suffix );
									?>
								</span>
```

(This gets rid of the extra `;` on line 1304 and formatting it this way with the opening/closing PHP tags on their own lines silences the PHPCS warnings.)

Reply:
Nice, thanks Barry.